docs/README.md

```diff
@@ -1,3 +1,13 @@
 # Docs
 
+## install doc requirements
+
+```
+pushd ..
```

Comment:
maybe overkill and depends where this starts from. Would just have `pip install ".[docs]"`

Reply:
removed